api/middleware.go

```diff
@@ -17,6 +18,7 @@ import (
 func TopMiddleware(handler http.Handler, config config.AppConfig) http.Handler {
 	//nolint:varnamelen
 	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		log.Printf("[%s %s]", r.Method, r.URL.Path)
```

Comment:
is this for debugging ? If so, it should be removed.

Reply:
it's not for debugging, it's simply for logging.